public/src/latestDependencies/java/com/revenuecat/purchases/models/GooglePurchaseOption.kt

```diff
@@ -0,0 +1,15 @@
+package com.revenuecat.purchases.models
+
+import android.os.Parcelable
+import kotlinx.parcelize.Parcelize
+
+@Parcelize
+data class GooglePurchaseOption(
```

Comment:
We will need to add docs to all these but that can come later I think

Reply:
i wanna clean this up a bit and add unit tests too, so i'll add them!

Reply:
okay defining a purchaseoption was a lot trickier than i expected haha. i'm sure we will want to iterate on this, but i got something there.